.eslintrc.json

```diff
@@ -11,9 +11,7 @@
   },
   "extends": [
     "eslint:recommended",
-    "plugin:react/recommended",
-    "plugin:prettier/recommended"
-  ],
+    "plugin:react/recommended"  ],
```

Comment:
why is prettier removed here ?

Reply:
is was causing issues when running the app